test/local/verification_reminder_db_tests.js

```diff
@@ -0,0 +1,141 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+require('ass')
+var tap = require('tap')
+var test = tap.test
+var uuid = require('uuid')
+var proxyquire = require('proxyquire')
+var log = { trace: console.log, info: console.log }
+
+var config = require('../../config').getProperties()
+var TestServer = require('../test_server')
+var Token = require('../../lib/tokens')(log)
+var DB = require('../../lib/db')(
+  config.db.backend,
+  log,
+  Token.error,
+  Token.SessionToken,
+  Token.KeyFetchToken,
+  Token.AccountResetToken,
+  Token.PasswordForgotToken,
+  Token.PasswordChangeToken
+)
+
+var zeroBuffer16 = Buffer('00000000000000000000000000000000', 'hex')
+var zeroBuffer32 = Buffer('0000000000000000000000000000000000000000000000000000000000000000', 'hex')
+
+function createTestAccount() {
+  var account = {
+    uid: uuid.v4('binary'),
+    email: 'reminder' + Math.random() + '@bar.com',
+    emailCode: zeroBuffer16,
+    emailVerified: false,
+    verifierVersion: 1,
+    verifyHash: zeroBuffer32,
+    authSalt: zeroBuffer32,
+    kA: zeroBuffer32,
+    wrapWrapKb: zeroBuffer32,
+    acceptLanguage: 'bg-BG,en-US;q=0.7,ar-BH;q=0.3'
+  }
+
+  return account
+}
+
+var mockLog = require('../mocks').mockLog
+
+var moduleMocks = {
+  '../config': {
+    'get': function (item) {
+      if (item === 'verificationReminders') {
+        return {
+          rate: 1
+        }
+      }
+    }
+  }
+}
+
+var dbServer
+var dbConn = TestServer.start(config)
+  .then(
+    function (server) {
+      dbServer = server
+      return DB.connect(config[config.db.backend])
+    }
+  )
+
+test(
+  'create',
+  function (t) {
+    var thisMockLog = mockLog({
+      increment: function (name) {
+        if (name === 'verification-reminders.created') {
+          t.end()
+        }
+      }
+    })
+
+    dbConn.then(function (db) {
+      var account = createTestAccount()
+      var reminder = { uid: account.uid.toString('hex') }
+
+      var verificationReminder = proxyquire('../../lib/verification-reminders', moduleMocks)(thisMockLog, db)
+
+      return verificationReminder.create(reminder).then(
+          function () {
+            t.end()
```

Comment:
Not sure if you need the second `t.end()` here. I believe it should only be called once.